src/odemis/driver/piezomotor.py

```diff
@@ -94,26 +96,26 @@ def __init__(self, name, role, port, axes, inverted=None, **kwargs):
             Axis parameters:
                 axis_number (0 <= int <= 127): typically 1-3 for x-z, required
                 closed_loop (bool): True for closed loop (with encoder), default to True
-                encoder_resolution (float): encoder resolution in m/step
-                spc (float): motor steps per encoder count, default to value in non-volatile memory
-                limit_type (0 <= int <= 2): type of limit switch, 0: no limit, 1: active high, 2: active low, default 0
-                range (tuple of float): in m, default to (0, STROKE_RANGE)
+                encoder_resolution (float): number of encoder counts per meter, default to 1.22e-9
+                motorstep_resolution (float): number of motor steps per m, default to 5e-6
+                range (tuple of float): in m, default to STROKE_RANGE
                 speed (float): speed in m/s
+                timeout_factor (int): the z axis typically behaves differently since it's not attached directly
+                    to the controller. Instead, there is a connection to a different component, and the movement
+                    of this different component causes the movement of the z axis. A larger movement in the component
+                    only causes a smaller movement in the z axis. For the most part, this is dealt with by adjusting
+                    the encoder resolution. However, after adjusting the encoder resolution, the timeouts are
+                    no longer useful (because we're assuming to move only a small amount on the axis, but actually,
+                    the movement of the second component is much larger and therefore also takes longer).
+                    This parameter allows to multiply the timeouts by a constant factor (1 by default).
```

Comment:
I don't get that. The default timeout is computed based on the range and the speed. Both of these should be the "user-visible". The time to move around this should also be based on this. It doesn't matter if the motor is directly connected to the stage or connected via a leaver. We care about the final stage movement. 
I don't exactly understand how everything is connected, but it seems that just multiplying encoder_resolution and motorstep_resolution by the factor would suffice. 

In any case, as a separate note, it's best to avoid describing the direct issue you're trying to solve in the docstring, but instead just describe the effect of the parameter. For instance here, the fact that timeout_factor is going to be used for the z axis in the current system has relationship with the actual effect. This parameter can be used on any axis. You should explain the issue and the use of the parameters in the microscope file.